dex-it/src/test/scala/com/wavesplatform/it/sync/MatcherRestartTestSuite.scala

```diff
@@ -50,6 +78,18 @@ class MatcherRestartTestSuite extends MatcherSuiteBase {
 
       dex1.api.waitForOrderStatus(aliceOrder, Status.Cancelled)
       dex1.api.getOrderHistoryByPublicKey(alice).head.status shouldBe Status.Accepted.name
+
+      markup("check Bob's orders")
+      bobsOrders.filter(order => order.id != bobsOrderToFulfil.id && order.id != bobsOrderToCancel.id).foreach { order =>
+        dex1.api.getOrderStatus(order).status shouldBe HttpOrderStatus.Status.Accepted
+      }
+
+      dex1.api.getOrderStatus(bobsOrderToFulfil).status shouldBe HttpOrderStatus.Status.Filled
+      dex1.api.getOrderStatus(bobsOrderToCancel).status shouldBe HttpOrderStatus.Status.Cancelled
+
     }
   }
+
+  private def mkBobOrder() = mkOrder(bob, wctWavesPair, OrderType.SELL, Gen.choose(10, 500).sample.get, 2.wct * Order.PriceConstant)
```

Comment:
There is a chance, that Gen selects two equal values. In such cases we introduce a parameter `i` and:
* either increase an amount or a price: https://github.com/wavesplatform/matcher/blob/master/dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala#L73  
* or decrease TTL: https://github.com/wavesplatform/matcher/blob/master/dex-it/src/test/scala/com/wavesplatform/it/sync/CancelOrderTestSuite.scala#L306

Suggest to decrease TTL.

Reply:
probably we can even don't use Gen here and stick to determining values.